templates/page/page.tpl.php

```diff
@@ -132,6 +138,13 @@
   <nav id="ubc7-unit-menu" role="navigation" class="navbar expand">
     <div class="navbar-inner expand">
       <?php print $fluidcontainerstart; ?>
+        <?php if ($drawer_enabled): ?>
+          <!-- UBC Unit Drawer Menu Button -->
+          <button style="float:right;margin-right:17px;" class="hamburger visible-md visible-lg drawer-toggle--primary" id="drawer-button">
+            <span>Menu ☰</span>
```

Comment:
I like the hamburger icon but maybe it should be consistent with the other hamburger with `<span class="icon-bar"></span>`?